Chp12Lsn7/UnitTestProject/UnitTest.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Chp12Lsn7;
+using UserCommunication;
+
+namespace UnitTestProject
+{
+    [TestClass]
+    public class UnitTest
+    {
+        [TestMethod]
+        [ExpectedException(typeof(System.ArgumentException), "Invalid Input! Cannot enter a number less than or equal to zero.")]
```

Comment:
If you are going to put energy into a message related to a unit test, first and foremost put the energy into the name of the test itself.   So instead of:

    public void SqrtTest()

Do this:

    public void Program_CalcSquareRootWithNegativeInput_ThrowsException()

The above naming is horrible for "production code," but it is correct and the standard developers expect for unit test code.